webpack/webpack.config.js

```diff
@@ -9,7 +9,7 @@ const DEBUG = process.env.NODE_ENV !== 'production'
 const PUBLIC_PATH = `/${process.env.PUBLIC_PATH || ''}/`.replace('//', '/')
 
 const config = {
-  devtool: DEBUG ? 'eval' : false,
+  devtool: DEBUG ? 'eval-source-map' : false,
```

Comment:
We are talking about debug mode. Being able to debug the application is very important for me, but feel free to change this back.